synapse/models/inet.py

```diff
@@ -1364,6 +1400,9 @@ async def _onSetWhoisText(node, oldv):
                 'doc': 'An FQDN whois registration record.'}),
 
             ('inet:whois:email', ('comp', {'fields': (('fqdn', 'inet:fqdn'), ('email', 'inet:email'))}), {
+                'interfaces': (
+                    ('meta:observable', {'template': {'observable': 'email address'}}),
```

Comment:
```suggestion
                    ('meta:observable', {'template': {'observable': 'WHOIS e-mail address'}}),
```

Reply:
We should probably fix `email` below to be `e-mail` so we're consistent with `inet:email` definition.